lib/app/presentation/utils/text_styles.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:supa_app/app/presentation/utils/app_colors.dart';
+
+class TextStyles {
+  static const TextStyle _baseStyle = TextStyle(
+    fontFamily: 'Sen',
+    color: AppColors.black,
+  );
+  // Heading [24sp to 36sp]
+  static TextStyle heading1({Color? color}) => _baseStyle.copyWith(
```

Comment:
Same with this , no more subheader 1 and 2 just 1 body, header, subheader subtitle etc, then fprefix with medium,large or bold.